app/parse.py

```diff
@@ -8,8 +15,51 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELD = [field.name for field in fields(Quote)]
+
+
+def create_quote_object(soup: Tag) -> Quote:
+    return Quote(
+        text=soup.select_one(".text").text,
+        author=soup.select_one(".author").text,
+        tags=[tag.text for tag in soup.select(".tags > a")]
+    )
+
+
+def get_single_data(page_soup: BeautifulSoup) -> list:
+    quotes = page_soup.select(".quote")
+    return [create_quote_object(page) for page in quotes]
+
+
+def get_data() -> list:
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]